compose-common.yml

```diff
@@ -12,35 +12,51 @@ base:
     # --link some-rabbit:rabbit "just works"
     - DJANGO_CELERY_BROKER_URL=amqp://guest@rabbit
     - DJANGO_DATABASE_URL=postgis://postgres@database/postgres
-    - DJANGO_EMAIL_HOST=localhost
-    # Settings above likely need to be changed, depending on  the environment
+    - APP_PORT=8000
+    - APP_HOST=0.0.0.0
+
+    # Logging
+    - PYTHONUNBUFFERED=non-empty-string
+    - PYTHONIOENCODING=utf-8
+    - DJANGO_LOG_LEVEL=INFO
+    
+    # We enable Python hash randomization to protect Django from certain DoS attacks.
+    # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/#python-options
+    - PYTHONHASHSEED=random
+
+
+    # Settings only need to be changed, if docker-compose is being changed
+    # or not used at all.
     ##########################################################################
-    # WARNING: restarting the server changes the secret key and thus loggs out
-    # other users.
-    - DJANGO_SECRET_KEY=`< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c120`
-    - DJANGO_EMAIL_BACKEND=django.core.mail.backends.smtp.EmailBackend
-    # EMAIL
-    - DJANGO_DEFAULT_FROM_EMAIL='noreply osmaxx <noreply@osmaxx.hsr.ch>'
 
-    # SECURITY
-    - DJANGO_ALLOWED_HOSTS='[".osmaxx.hsr.ch.",]'
+    # port and host application listens
+    - APP_PORT=8000
+    - APP_HOST=0.0.0.0
 
-    # SSL
-    # set this to 60 seconds and then to 518400 when you can prove it works
-    - DJANGO_SECURE_HSTS_SECONDS=60
-    - DJANGO_SECURE_HSTS_INCLUDE_SUBDOMAINS=true
-    - DJANGO_SECURE_SSL_REDIRECT=true
-    - DJANGO_SECURE_SSL_HOST='osmaxx.hsr.ch'
-    - DJANGO_SECURE_REDIRECT_EXEMPT=[]
+    # EMAIL
+    - DJANGO_EMAIL_BACKEND=django.core.mail.backends.smtp.EmailBackend
+    - DJANGO_EMAIL_HOST=localhost
+    - DJANGO_EMAIL_HOST_PASSWORD=
+    - DJANGO_EMAIL_HOST_USER=
```

Comment:
If we leave the out in the setting YMLs, they should also be unset, shouldn't they?